www/content/tutorial.md

```diff
@@ -859,6 +858,27 @@ Here's what this does:
 
 Thanks to the `?` postfix operator, your code can focus on the "happy path" (where nothing fails) and simply bubble up to the caller any error that might occur. Your error handling code can be neatly separated, and you can rest assured that you won't forget to handle any errors, since the compiler will let you know. See this [code example](https://www.roc-lang.org/examples/Results/README.html) for more details on error handling.
 
+### [Recovering from errors with the `??` infix operator](#recovering-from-errors) {#recovering-from-errors}
+
+Above you saw this code sample to show how to recover from errors using the
+`Result.with_default` function:
+
+```roc
+Result.with_default(List.get(["a", "b", "c"], 100), "")
+# returns "" because that's the default we said to use if List.get returned an Err
+```
+
+But this was common enough that Roc has syntax to make it easier to read and
+write:
+
+```roc
+List.get(["a", "b", "c"], 100) ?? ""
```

Comment:
Consider replacing all `Result.with_default` usages after this line with ??

Reply:
Showing both